README.md

```diff
@@ -112,12 +112,31 @@ Then using our Addons component in ChatScreen
 ```javascript
 import React from 'react'
 import {ChatScreen as BaseChatScreen} from 'rn-firebase-chat'
-import {CameraView, useCamera} from 'rn-firebase-chat/src/addons/camera'
+import {
+  CameraView,
+  useCamera,
+  FileAttachmentRef,
+  FileAttachment,
+} from 'rn-firebase-chat/src/addons';
 
 ...
 
 export const ChatScreen: React.FC = () => {
   const {onPressCamera, onPressGallery} = useCamera()
+  const fileAttachmentRef = useRef<FileAttachmentRef>(null);
+  const renderLeftCustomView = useCallback(() => {
+    return (
+      <View>
+        <Button
```

Comment:
Is there an other way to handle rendering document button like camera and gallery buttons? I think we should use one style for all addons